pymc/distributions/shape_utils.py

```diff
@@ -674,3 +675,8 @@ def maybe_resize(
         )
 
     return rv_out
+
+
+def rv_size_is_none(size: Variable) -> bool:
+    """Check wether an rv size is None (ie., at.Constant([]))"""
+    return isinstance(size, Constant) and size.data.size == 0
```

Comment:
It's perfectly fine to do `pm.get_moment(..., size=())`, right? Or even with `size=[]`?
Maybe this utility function should account for `size` being `()`, `[]` or `None` too?

Reply:
No, the size comes from the RV and is always converted to an empty string when it's None IIRC